plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/OpenDropDownMenuHandler.java

```diff
@@ -0,0 +1,107 @@
+
+package com.google.cloud.tools.eclipse.ui.util;
+
+import com.google.common.base.Strings;
+import org.eclipse.core.commands.AbstractHandler;
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+import org.eclipse.jface.action.ContributionItem;
+import org.eclipse.jface.action.IMenuListener2;
+import org.eclipse.jface.action.IMenuManager;
+import org.eclipse.jface.action.MenuManager;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.graphics.Point;
+import org.eclipse.swt.graphics.Rectangle;
+import org.eclipse.swt.widgets.Event;
+import org.eclipse.swt.widgets.Menu;
+import org.eclipse.swt.widgets.ToolItem;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.menus.IMenuService;
+
+/**
+ * A command handler for commands representing drop-down Menu contribution item. This handler just
+ * opens drop-down menu, since Eclipse behavior for such items is to open drop-down menu only by
+ * clicking on small arrow. Drop-down Menu contribution items without any handler shown as disabled.
```

Comment:
shown --> are shown
Is that true for drop-down menu contribution items that are not themselves action buttons?